experiments/PetSet/PetSet.jl

```diff
@@ -0,0 +1,104 @@
+using Match
+using Catlab.Meta, Catlab.Syntax
+using Catlab.Graphics.Graphviz
+import Catlab.Graphics.Graphviz: Graph, Edge
+using Petri
+using LabelledArrays
+
+using Catlab
+using Catlab.Present
+using Catlab.Theories
+using Catlab.CategoricalAlgebra.CSets
+
+@present SITOS(FreeCategory) begin
+  species::Ob
+  inputs::Ob
+  transitions::Ob
+  outputs::Ob
+
+  is::Hom(inputs,species)
+  it::Hom(inputs,transitions)
+  os::Hom(outputs,species)
+  ot::Hom(outputs,transitions)
+end
+
+PetSet = CSetType(SITOS, index=[:is,:it,:os,:ot])
+
+nspecies(ps::PetSet) = nparts(ps,:species)
+ntransitions(ps::PetSet) = nparts(ps,:transitions)
+ninputs(ps::PetSet) = nparts(ps,:inputs)
+noutputs(ps::PetSet) = nparts(ps,:outputs)
+
+# Need a better name for this
+function from_std_petri(S,T)
```

Comment:
We could also probably have another method call along the lines of `from_std_petri(m::Petri.Model) = from_std_petri(m.S, m.Δ)`